util.c

```diff
@@ -135,8 +135,9 @@ mkstr(size_t n)
 }
 
 void
-delevalstr(struct evalstring *str)
+delevalstr(void *pn)
```

Comment:
I think we should come up with a consistent  name for this parameter. Perhaps `ptr` or `data`?

Reply:
I'll use ptr